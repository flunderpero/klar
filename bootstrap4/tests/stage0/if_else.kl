fn main():
    let x = 5
    if x == 5:
        print(5)
    end
    if x == 6:
        print(6)
    else:
        print(5)
    end
    return 0
end
