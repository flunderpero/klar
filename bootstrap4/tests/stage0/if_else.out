5
5
---
exit code: 0
